elastica/contact_forces.py

```diff
@@ -0,0 +1,77 @@
+__doc__ = """ Numba implementation module containing contact between rods and rigid bodies and other rods rigid bodies or surfaces."""
+
+from elastica.typing import SystemType, AllowedContactType
+from elastica.rod import RodBase
+
+
+class NoContact:
+    """
+    This is the base class for contact applied between rod-like objects and allowed contact objects.
+
+    Notes
+    -----
+    Every new contact class must be derived
+    from NoContact class.
+
+    """
+
+    def __init__(self):
+        """
+
+        Parameters
+        ----------
+
+        """
+
+    def _order_check(
+        self,
+        system_one,
+        system_two,
+    ):
```

Comment:
add type hints, since you already have in docstring

Reply:
ok